client/src/pages/HomePage/LatestArrival.css

```diff
@@ -0,0 +1,46 @@
+/* Container  */
+
+.container-latest-arrival {
+  display: flex;
+  height: 60vh;
+  background-color: rgba(141, 106, 87, 1);
+  margin: 6% 8% 6% 8%;
```

Comment:
C'est une mauvaise idée de long terme que d'avoir des unités relatives sur les marges. Je ne demande pas de changement en l'état, mais attention aux surprise que ça pourrait créer.

Reply:
C'est noté je ferai attention à l'avenir et je modifierai si besoin